### ISSUE TITLE
Can the field be used in the frontend?

### ISSUE THREAD

**Issue by Taitava** [Firesphere/silverstripe-rangefield#7]
Hi,

I just tried to use the range field in the frontend instead of the backend (= cms/admin side). One thing I noticed was that the CSS and JS files were not included automatically, so I had to do this in my controller class:

```php
protected function init()
{
	parent::init();

	Requirements::css('firesphere/rangefield:client/dist/main.css');
	Requirements::javascript('firesphere/rangefield:client/dist/main.js');
}
```
(Maybe this could be added to the readme file?)

But that doesn't make the field usable yet, as it now appears like this in my layout:

![image](https://user-images.githubusercontent.com/13002142/76701064-45461500-66c6-11ea-85c1-7dfd5dc19dc1.png)

*Hinta* is the name of the range field. The field just appears as a gray vertical line. It's entirely possible that my own layout / css rules may make it look like that. I just don't have the time to inspect that right now, will do it later.

So in summary, I would just like to ask that should it be generally possible to use this field outside of the admin editor? Thanks for support! :)

**Comment by Firesphere** [comment]
As far as I am aware, it should be useable in the frontend.

Do you see any errors in the console?

**Comment by Taitava** [comment]
Thanks for your answer :). Now that I tried this again, I started from scratch, reinstalled the module (as I had temporarily uninstalled it), and rewrote the call to create the field. I did happen to get it working just by tinkering with the parameters I submitted to the constructor call. So there was no issue with the module itself :).

I had some errors in the browser's console but only because I accidentally submitted an associative array as the `$start` parameter to the constructor. But now I don't have any errors anymore in the browser's console.

I have opened [a pull request](https://github.com/Firesphere/silverstripe-rangefield/pull/16) for just a small addition to the Readme.md file that covers defining the CSS and JavaScript requirements when using this field in the frontend. What do you think about this? Of course you can reword it if you want :).

**Comment by Firesphere** [comment]
PR accepted, cheers for your work!
